.circleci/config.yml

```diff
@@ -49,7 +52,23 @@ aliases:
        pip install selenium
        pip install pyvirtualdisplay
        pip install nose
```

Comment:
doesn't `testsrunner` bring in `nose`?

Reply:
you are right, let me remove it and validate, and will merge. thanks.